### PR TITLE
changed alert message on wiki creation

### LINKED ISSUE #4504: Show different alert at wiki creation if work is in moderation
Currently, new users are not allowed to create a wiki page and shown this alert - https://github.com/publiclab/plots2/issues/4463#issuecomment-450570884

If the user's work is in moderation we can show them different alert may be saying "You can create the wiki once your research note/question is approved by moderators."

### PR DESCRIPTION
**Pull request by Hassanelashram**
Fixes #4504 

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] PR is descriptively titled 📑 and links the original issue above 🔗
* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts 📁
* [x] screenshots/GIFs are attached 📎 in case of UI updation
* [x] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software 

Thanks!

### CODE REVIEW

**Comment by Hassanelashram** [comment]
@gauravano Could you please review ?

**Comment by gauravano** [comment]
Hi @Hassanelashram, thank you for opening the PR. The current solution requires improvement. A new user can post to publiclab.org only if they have one approved node (question/research note).

Now, there can be 2 cases: 
1. User has not created any note till now
2. User has created note(s) but it's still in review   

Currently, we don't differentiate between the above 2 cases and show this message -  _Please post a question or other content before editing the wiki. Click <a href='https://publiclab.org/notes/tester/04-23-2016/new-moderation-system-for-first-time-posters'>here</a> to learn why._

Now, either we can start differentiating with an extra condition to check if there is any post in review for the current user which will make one more DB query or we can just append in the current message to say that _Please post a question or other content before editing the wiki. Click <a href='https://publiclab.org/notes/tester/04-23-2016/new-moderation-system-for-first-time-posters'>here</a> to learn why. If you have already posted a note then you would be able to edit the wiki as soon as your post get approved by moderators_ -- something like this :)

@jywarren @cesswairimu what do you think would be better?

**Comment by cesswairimu** [comment]
I think differentiating between the two and giving different flash messages would be ideal

**Comment by jywarren** [comment]
Sorry, just trying to retrigger the tests now that we've resolved an unrelated issue!

**Comment by cesswairimu** [comment]
I think this might need a rebase Travis still shows up after reopening

**Comment by jywarren** [comment]
Rebased!

File: test/functional/wiki_controller_test.rb [hunk 1]
```diff
@@ -193,7 +193,7 @@ def teardown
         id: 'chicago'
         }
 
-    assert_equal flash[:notice], "Please post a question or other content before editing the wiki. Click <a href='https://publiclab.org/notes/tester/04-23-2016/new-moderation-system-for-first-time-posters'>here</a> to learn why."
+    assert_equal flash[:notice], "You can create the wiki once your research note/question is approved by moderators. Click <a href='https://publiclab.org/notes/tester/04-23-2016/new-moderation-system-for-first-time-posters'>here</a> to learn why."
```
Comment (jywarren) [inline test/functional/wiki_controller_test.rb:196]: So strange that this failed!? I don't understand -- we changed the test here too!

```
Progress: |=== FAIL WikiControllerTest#test_disallow_viewing_edit_wiki_page_for_first-timers (38.87s)
        --- expected
        +++ actual
        @@ -1 +1 @@
        -"Please post a question or other content before editing the wiki. Click <a href='https://publiclab.org/notes/tester/04-23-2016/new-moderation-system-for-first-time-posters'>here</a> to learn why."
        +"You can create the wiki once your research note/question is approved by moderators. Click <a href='https://publiclab.org/notes/tester/04-23-2016/new-moderation-system-for-first-time-posters'>here</a> to learn why."
        test/functional/wiki_controller_test.rb:196:in `block in <class:WikiControllerTest>'
```

Comment (jywarren) [reply test/functional/wiki_controller_test.rb:196]: Aha! Ok, we actually have to make the change twice in the controller. I'll make a suggestion!!!

**Comment by jywarren** [comment]
OK, my apologies, there were a couple extra places to make the same change! I was able to add the extra changes and with luck this will now pass. Thank you for your patience!!

**Comment by jywarren** [comment]
Excellent, thank you so much!!!

**Pull request merged**